dcicutils/misc_utils.py

```diff
@@ -1518,6 +1518,19 @@ def __repr__(self):
         return f"<{self.name}@{id(self):x}>"
 
 
+def key_value_pair(key, value):
+    return {'Key': key, 'Value': value}
```

Comment:
```suggestion
def key_value_pair(key, value):
    """
    Given a key k and a value v, returns the dictionary {"Key": k, "Value": v}, which is a format AWS is fond of.
    """
    return {'Key': key, 'Value': value}
```

I would be OK with a name change here, which might make @willronchetti happier, perhaps calling it `key_value_dict` instead of `key_value_pair`. @alexander-veit, if you're using PyCharm, that's an easy change to make. If you're not, I could make that change separately and upload the fix.